jekyll/_cci2/env-vars.md

```diff
@@ -80,21 +101,68 @@ Login Succeeded
 Not all command-line programs take credentials
 in the same way that `docker` does.
 
-## Adding Global Environment Variables
+## Setting Context-Level Environment Variables
 
-To add global environment variables that may be shared across projects, use the Settings > Contexts page of the CircleCI application. See the [Contexts]( {{ site.baseurl }}/2.0/contexts/) documentation for instructions.
```

Comment:
Don't remove this information please.

Reply:
Which information specifically? I think the only difference in content is the signposting for "Settings > Contexts".